std/algorithm/iteration.d

```diff
@@ -39,6 +39,9 @@ $(T2 permutations,
         Lazily computes all permutations using Heap's algorithm.)
 $(T2 reduce,
         $(D reduce!((a, b) => a + b)([1, 2, 3, 4])) returns $(D 10).)
+$(T2 scan,
+        $(D scan!((a, b) => a + b)([1, 2, 3, 4])) lazily returns a range containing
```

Comment:
Maybe "…returns a lazily-evaluated range…"? Maybe it's just me, but "lazily returns a range" sounds a little weird.